.env.example

```diff
@@ -1 +1,5 @@
-BOT_TOKEN = ''
\ No newline at end of file
+BOT_TOKEN = ''
+
+REDIS_HOST = 'redis' # «redis» for Docker build, «localhost» for local usage
+REDIS_PORT = 6379
+REDIS_DB = 0
```

Comment:
consider not using all redis related data in env, because these parameters definitely cannot change